### ISSUE TITLE
Insufficient contrast in DatePicker for days adjacent to current month

### ISSUE THREAD

**Issue by rstrangfeld** [IBM/carbon-components-react#2124]
In the DatePicker, when displaying a month, the ending days of the previous month and beginning days of the next month are shown in light gray at the top and bottom of the calendar page. They are still clickable like the days in the current month.

DAP reports contrast violations for these light gray days, with the message:
"Minimum contrast of 2.11 with its background is not sufficient to meet WCAG AA requirements for text of size 12px and weight of 400."

The relevant WCAG criterion is Success Criterion 1.4.3 Contrast (Minimum).

This was found using carbon-components-react 6.71.0 and carbon-components 9.62.3.

**Comment by rstrangfeld** [comment]
It looks like stale bot is threatening to close this issue because of inactivity. We're still interested in a fix. What would need need to happen for that to come about?
Thanks!

**Comment by dakahn** [comment]
Definitely fixable for sure. Having proper contrast on _all the dates_ is important. Our current v10 implementation seems to have adjusted the styling for better contrast (though the shape of the month is lost impacting _usability_).

![2019-05-03 12_18_09-Carbon Components React](https://user-images.githubusercontent.com/40970507/57154058-89297780-6d9d-11e9-9f37-d1cfd5754cc3.png)
![2019-05-03 12_21_17-Carbon Design System](https://user-images.githubusercontent.com/40970507/57154228-f89f6700-6d9d-11e9-9065-7c09f8356f8b.png)
